Tutorials_UsingAPIs/Flow-Control-EventQueue/main.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2020 Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "mbed.h"
+#include "mbed_events.h"
+
+DigitalOut led1(LED1);
```

Comment:
What is the reason for having these two objects as global? Why are they not created within `main()`?

Reply:
I think in Mbed OS applications it's common to put "hardware" objects like buttons, LEDs, network interface etc. in static memory, as in most examples.

Reply:
It is still generally considered bad practice though. We should not encourage that.

Reply:
For this example I can move them into `main()`. Though it'll make it inconsistent with other examples/snippets we have.

Reply:
Done